docs/platform/block-storage/how-to-use-block-storage-with-your-linode-classic-manager/index.md

```diff
@@ -143,6 +143,24 @@ Storage volumes **cannot** be sized down, only up. Keep this in mind when sizing
 
         mount /dev/disk/by-id/scsi-0Linode_Volume_BlockStorage1 /mnt/BlockStorage1
 
+## How to Transfer a Volume to a New Linode
+
+1. Follow the steps to safely detach your volume as mentioned [above]
+(#how-to-detach-a-block-storage-volume-from-a-linode).
+
+1.  Select the label of the Linode you want to attach the volume to. Then click **Attach**:
+
+    [![Linode Manager add volume](bs-volume-attach-small.png)](bs-volume-attach.png)
+
+1.   The Volume already has a filesystem on it, so we just need to create a mountpoint for the new Linode, provided it hasn't already been created:
```

Comment:
Let's stay away from "we" here, instead maybe something like "...so all that is needed is to create a mountpoint..."